.github/pull_request_template.md

```diff
@@ -23,3 +23,11 @@ Please update the following to link the associated issue. This is required for s
 -->
 
 Fixes: #issue_number
+
+## Contributor checklist
+Please verify that the PR meets the following requirements, where applicable:
+
+- [ ] An overview of proposed schema changes is included in a linked GitHub issue.
+- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
```

Comment:
should we combine these? they either update the GH issue with the details on the schema changes or they do a design doc if it involves new API changes

Reply:
Could you suggest the user facing phrasing you have on mind for combining these?